### PR TITLE
[bitnami/testlink] MAJOR: Follow non-root approach

### PR DESCRIPTION
**Pull request by andresbono**
Signed-off-by: Andrés Bono <andresbono@vmware.com>

**Description of the change**

We are working on releasing the TestLink Docker image with a non-root approach. As part of that, we need to update the chart to use the new port numbers used by the container image and related settings.

We're also taking advantage of this PR being a major change by adding all standarizations (i.e. customizable command, extra volumes, extra envs, K8s labels standarization, etc.).

**Benefits**

Support for environments such as OpenShift, where a non-root UID is required to run a container.

**Possible drawbacks**

Breaking change with previous chart versions, as the default HTTP/HTTPS ports are changed to 8080/8443 (previously 80/443). The labels may also cause this to be a breaking change.

**Applicable issues**

None

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/testlink/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ $ helm install my-release bitnami/testlink
 
 ## Introduction
 
-This chart bootstraps a [TestLink](https://github.com/bitnami/bitnami-docker-testlink) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+This chart bootstraps an [TestLink](https://github.com/bitnami/bitnami-docker-testlink) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
```
Comment (marcosbc) [inline bitnami/testlink/README.md:14]: an TestLink -> a TestLink

**Review by marcosbc — APPROVED** [review]
LGTM with comment(s)

File: bitnami/testlink/README.md (hunk 1 above)
Comment (andresbono) [reply bitnami/testlink/README.md]: 🦅

**Review by marcosbc — APPROVED** [review]
LGTM, thanks!

**Pull request merged**